clients/vulnerabilities.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/ossf/scorecard/v3/errors"
+)
+
+const osvQueryEndpoint = "https://api.osv.dev/v1/query"
+
+type osvQuery struct {
+	Commit string `json:"commit"`
+}
+
+// VulnerabilitiesClient cheks for vulnerabilities in api.osv.dev.
+type VulnerabilitiesClient interface {
+	HasUnfixedVulnerabilities(context context.Context, commit string) (OSVResponse, error)
+}
+
+// OSVResponse is the response from the OSV API.
+type OSVResponse struct {
```

Comment:
Since this is a response from an interface and not from a specific implementation of the interface, rename to something like `VulnerabilitiesResponse`.

Reply:
Agreed. Thanks!